build/config.props

```diff
@@ -35,15 +35,17 @@
 
   <!-- Nuspec defaults -->
   <PropertyGroup>
-    <Authors>NuGet</Authors>
-    <ProjectUrl>https://github.com/NuGet/NuGet.Client</ProjectUrl>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/NuGet/NuGet.Client/dev/LICENSE.txt</PackageLicenseUrl>
+    <Authors>Microsoft</Authors>
+    <PackageProjectUrl>https://aka.ms/nugetprj</PackageProjectUrl>
+    <PackageLicenseUrl>https://aka.ms/nugetlicense</PackageLicenseUrl>
```

Comment:
Should the license be embedded?

Reply:
The tooling is not there on the CI machines. 

The latest(5.0) already use license expressions.